meson.build

```diff
@@ -13,7 +13,7 @@ deps = [
     dependency('gee-0.8'),
     dependency('gstreamer-1.0'),
     dependency('gstreamer-base-1.0'),
-    dependency('gtksourceview-3.0'),
```

Comment:
I think we should make gtksourceview-4 a fallback dependency at least. There's no need to bump up the version requirement.

Reply:
> There's no need to bump up the version requirement

gtksourceview-3 doesn't compile on the modern supported gnome platform 3.38, but I made gtksouceview-4 a fallback